core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java

```diff
@@ -17,8 +17,10 @@
 import tc.oc.pgm.api.match.MatchModule;
 import tc.oc.pgm.api.match.MatchScope;
 import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.events.ListenerScope;
 import tc.oc.pgm.util.nms.NMSHacks;
 
+@ListenerScope(value = MatchScope.LOADED)
```

Comment:
I think this should actually just be `RUNNING`.

Reply:
But if we do this, we need to clean up `entities` when the match is finished. (we can't guarantee the recurring task will execute one last time)

Reply:
Why should this not run before/after a match? e.g seeing where your TNT _would_ have exploded right after a match ends seems reasonable.

Reply:
It's just unnecessary.. a lot of stuff can happen post-cycle, sending a more fake packets to clients for no reason isn't a good idea.

Reply:
Ok, ill move it to `Running` and figure out the cleaning thing

Reply:
I also think it should stay on LOADED, match not running doesn't mean tnt can't still be moving.